lms/envs/bok_choy.env.json

```diff
@@ -79,7 +79,6 @@
         "ENABLE_INSTRUCTOR_ANALYTICS": true,
         "ENABLE_S3_GRADE_DOWNLOADS": true,
         "ENABLE_THIRD_PARTY_AUTH": true,
-        "ENABLE_COMBINED_LOGIN_REGISTRATION": true,
```

Comment:
I guess I was naively hoping that we'd be completely removing this feature flag in the near future :-p

I will push a little reversal of the changes to the test envs so the flag is there explicitly. I think you make a good point.